### PR TITLE
Test improvements while attempting to publish Scala 2.11 support

### PR DESCRIPTION
**Pull request by jeffmay**
@htmldoug @kionka-rally @paul-a-kennedy-rally @sklei2 

Publish `play-test-ops_2.12`
~~Publish `play-test-ops_2.13`~~

UPDATE: The correct version was being published, however, I made a few changes on this branch that might be worth getting in, such as adding back support for Scala 2.12

### CODE REVIEW

**Review by usufruct99 — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -71,7 +75,6 @@ def coreProject(includePlayVersion: String): Project = {
   val path = s"play$playSuffix-core"
   commonProject(path, path).settings(
     name := s"play$playSuffix-test-ops-core",
-    scalaVersion := scalaVersions.head,
```
Comment (jeffmay) [inline build.sbt:74]: I'm not certain this was broken to begin with, now that I double check it:
https://bintray.com/rallyhealth/maven/play27-test-ops-core/1.1.3#files/com%2Frallyhealth%2Fplay27-test-ops-core_2.13%2F1.1.3

File: build.sbt [hunk 2]
```diff
@@ -20,7 +20,7 @@ ThisBuild / licenses += ("MIT", url("https://opensource.org/licenses/MIT"))
   */
 val suppressSemVerCheckOfNewScalaVersionsUntilNextVersion = semVerCheck := {
```
Comment (jeffmay) [inline build.sbt:25]: I can remove this whole thing now, I believe

File: build.sbt [hunk 3]
```diff
@@ -13,30 +13,33 @@ ThisBuild / bintrayRepository := "maven"
...
 /**
   * Semantic versioning attempts to validate that the version generated makes sense relative to previous
   * versions released. We are introducing support for new Scala versions in this release, so the semVerCheck
   * will fail. This setting will ensure that we don't forget to re-enable it after this release.
   */
 val suppressSemVerCheckOfNewScalaVersionsUntilNextVersion = semVerCheck := {
   version.value match {
-    case VersionNumber(Seq(1, 1, x, _*), _, _) if x <= 3 => Def.task {}
+    case VersionNumber(Seq(1, 1, 3 | 4, _*), _, _) => Def.task {}
+    case VersionNumber(Seq(1, 2, 0, _*), _, _) => Def.task {}
```
Comment (danny-goldberg-rally) [inline build.sbt:28]: if you're next releasing 1.1.4, why suppress semver for scala 2_12 for 1.2.0 as well?

Comment (jeffmay) [reply build.sbt]: I plan to release it as 1.2.0. Not sure why I added both.

Comment (jeffmay) [reply build.sbt:28]: Not sure whether I should release as 1.1.4 or 1.2.0

**Review by danny-goldberg-rally — APPROVED** [review]

**Pull request merged**